code/robots.py

```diff
@@ -101,6 +102,20 @@ def consolidate(field: str, value: str) -> str:
     return sorted_robots
 
 
+def clean_robot_name(name):
+    """ Clean the robot name by removing some characters that were mangled by html software once. """
+    # This was specifically spotted in "Perplexity-User"
+    # Looks like a non-breaking hyphen introduced by the HTML rendering software
+    # Reading the source page for Perplexity: https://docs.perplexity.ai/guides/bots
+    # You can see the bot is listed several times as "Perplexity‑User" with a normal hyphen, 
```

Comment:
Best not to use a non-breaking hyphen here!
```suggestion
    # You can see the bot is listed several times as "Perplexity-User" with a normal hyphen, 
```